packages/hash/external-services/kratos/kratos.dev.yml

```diff
@@ -78,6 +78,16 @@ selfservice:
       ui_url: http://localhost:3000/verification
       enabled: true
 
+    recovery:
+      enabled: true
+      # Set through SELFSERVICE_FLOWS_RECOVERY_UI_URL
+      ui_url: http://localhost:3000/recovery
+      use: code
+
+    settings:
+      # Set through SELFSERVICE_FLOWS_SETTINGS_UI_URL
+      ui_url: http://localhost:3000/settings
```

Comment:
Note that I've not updated the production kratos config, perhaps we should leave this for a separate PR so that it can be reviewed by @thehabbos007 and deployed before that PR is merged? In which case I'll create a task

Reply:
cc @CiaranMn @nonparibus

Reply:
Captured and assigned to @benwerner01 ([internal link](https://app.asana.com/0/1203363157432094/1203604956440372/f)) -- feel free to reassign to @thehabbos007 if agreed between yourselves.